fallback.c

```diff
@@ -6,6 +6,7 @@
 
 #include <efi.h>
 #include <efilib.h>
+#include <CrtLibSupport.h>
```

Comment:
This looks incorrect, and it is breaking the CI build - but just removing it works fine.

Reply:
Looks like we need `OpenSslSupport.h` there, for `isxdigit`.